### PR TITLE
Upcoming demo 052022

### PR DESCRIPTION
**Pull request by kpodemski**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Please be specific when describing the PR. What did you add, why did you submit it?
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

**Review by matthieu-rolland — APPROVED** [review]

File: news/_posts/public-oss-demo/2022-07-01-upcoming-demo-5-2022.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title:  "Announcing July PrestaShop Public OSS Demo"
+subtitle: "Meet us for the fifth session of the open source project's public demo in 2022"
+date:   2022-07-01
```
Comment (matks) [inline news/_posts/public-oss-demo/2022-07-01-upcoming-demo-5-2022.md]:
```suggestion
date:   2022-07-04 10:00:00
```

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @kpodemski

**Comment by matks** [comment]
Failed deploy https://build.prestashop.com/

😭 😭 😭 😭 😭 😭 😭 😭

**Comment by matks** [comment]
Fixed

**Pull request merged**